airflow/utils/db.py

```diff
@@ -28,6 +28,7 @@
 import contextlib
 
 from airflow import settings
+from airflow.models.knowneventtype import KnownEventType
```

Comment:
Can you change the filename to `known_event_type` instead of `knowneventtype` as it is more readable

Reply:
Sure. For consistency, I think it would then be best for all /airflow/models/*.py files to be named the same as the SQLAlchemy table name which are all written with underscores, so I'll stick to that if that's okay (since I'll [move more classes from models.py](https://issues.apache.org/jira/issues/?jql=text%20~%20%22Refactor%3A%20Move%20out%20of%20models.py%22%20AND%20reporter%20in%20(Fokko%2C%20BasPH))).